src/components/input/input-base.ts

```diff
@@ -130,12 +138,6 @@ export abstract class IgcInputBaseComponent extends SizableMixin(
     selectMode: 'select' | 'start' | 'end' | 'preserve' = 'preserve'
   ) {
     this.input.setRangeText(replacement, start, end, selectMode);
-
-    if (this.value !== this.input.value) {
-      this.value = this.input.value;
```

Comment:
Just implement the input value assignment in the `igc-input` component as you have a failing test currently.